settings.gradle.kts

```diff
@@ -25,54 +25,75 @@ dependencyResolutionManagement {
+include(":extensions:opentelemetry-extension-annotations")
+include(":extensions:opentelemetry-extension-incubator")
+include(":extensions:opentelemetry-extension-aws")
+include(":extensions:opentelemetry-extension-kotlin")
+include(":extensions:opentelemetry-extension-noop-api")
+include(":extensions:opentelemetry-extension-trace-propagators")
+include(":exporters:opentelemetry-exporter-jaeger")
+include(":exporters:opentelemetry-exporter-jaeger-thrift")
+include(":exporters:opentelemetry-exporter-logging")
+include(":exporters:opentelemetry-exporter-logging-otlp")
+include(":exporters:otlp:opentelemetry-exporter-otlp")
+include(":exporters:otlp:opentelemetry-exporter-otlp-common")
+include(":exporters:otlp:opentelemetry-exporter-otlp-metrics")
+include(":exporters:otlp:opentelemetry-exporter-otlp-trace")
+include(":exporters:otlp-http:opentelemetry-exporter-otlp-http-metrics")
+include(":exporters:otlp-http:opentelemetry-exporter-otlp-http-trace")
+include(":exporters:opentelemetry-exporter-prometheus")
+include(":exporters:opentelemetry-exporter-zipkin")
 include(":integration-tests")
 include(":integration-tests:tracecontext")
-include(":opencensus-shim")
-include(":opentracing-shim")
+include(":opentelemetry-opencensus-shim")
+include(":opentelemetry-opentracing-shim")
 include(":perf-harness")
-include(":proto")
-include(":sdk:all")
-include(":sdk:common")
-include(":sdk:metrics")
-include(":sdk:metrics-testing")
-include(":sdk:testing")
-include(":sdk:trace")
-include(":sdk:trace-shaded-deps")
-include(":sdk-extensions:async-processor")
-include(":sdk-extensions:autoconfigure")
-include(":sdk-extensions:autoconfigure-spi")
-include(":sdk-extensions:aws")
-include(":sdk-extensions:logging")
-include(":sdk-extensions:resources")
-include(":sdk-extensions:tracing-incubator")
-include(":sdk-extensions:jaeger-remote-sampler")
-include(":sdk-extensions:jfr-events")
-include(":sdk-extensions:zpages")
+include(":opentelemetry-proto")
+include(":sdk:opentelemetry-sdk")
+include(":sdk:opentelemetry-sdk-common")
+include(":sdk:opentelemetry-sdk-metrics")
+include(":sdk:opentelemetry-sdk-metrics-testing")
+include(":sdk:opentelemetry-sdk-testing")
+include(":sdk:opentelemetry-sdk-trace")
+include(":sdk:sdk-trace-shaded-deps")
+include(":sdk-extensions:opentelemetry-sdk-extension-async-processor")
+include(":sdk-extensions:opentelemetry-sdk-extension-autoconfigure")
+include(":sdk-extensions:opentelemetry-sdk-extension-autoconfigure-spi")
+include(":sdk-extensions:opentelemetry-sdk-extension-aws")
+include(":sdk-extensions:opentelemetry-sdk-extension-logging")
+include(":sdk-extensions:opentelemetry-sdk-extension-resources")
+include(":sdk-extensions:opentelemetry-sdk-extension-tracing-incubator")
+include(":sdk-extensions:opentelemetry-sdk-extension-jaeger-remote-sampler")
+include(":sdk-extensions:opentelemetry-sdk-extension-jfr-events")
+include(":sdk-extensions:opentelemetry-sdk-extension-zpages")
+
+// Map project names to directory names.
```

Comment:
I added this instead of just renaming the folders for now. I would prefer renaming the folders but that makes the PR *huge* (1070 changed files). Can do in a follow-up.